src/plugins/vis_augmenter/common/constants.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const APP_PATH = {
+  STATS: '/stats',
+};
+export const APP_API = '/api/vis_augmenter';
+
+// used for limiting results received from the stats API
+export const PER_PAGE_REQUEST_NUMBER = 50;
```

Comment:
How come this doesn't take from

```
savedObjects:perPage
```

Reply:
I'm not familiar with where that is used or in what context. For this situation, it's just used for collecting all of the saved objects quickly to provide aggregate results in the `stats` api. I had used this value which is used in the same context in a different plugin, and confirmed with them this value works fine.

Can you point me to where this is used? If just the saved objects list page, prefer to leave this value since I would assume the setting value is probably smaller (10, 20, etc.)

Reply:
It's a UI Setting and can be accessed from the saved objects start plugin like:

```
SavedObjectsStart.settings.getPerPage()
```

It defaults to 20 but can be configurable. Could be a quick win to make your stats API more configurable

Reply:
Since this is all defined in `start` phase, but we register the routes and need the value at the `setup` phase, I've simplified things by exposing the default values set in the saved_objects plugin in the [latest commit](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4503/commits/02e56a3a510d785689b486adbf66ad868f7f6a4e). Lmk what you think.